### PR TITLE
Make sample results fully visible without the need for scrolling

### PR DESCRIPTION
**Pull request by DelazJ**
https://docs.qgis.org/testing/en/docs/server_manual/containerized_deployment.html#first-try last code block now renders as below
Capture d’écran 2020-09-09 à 13 55 28

### CODE REVIEW

File: docs/server_manual/containerized_deployment.rst [hunk 1]
```diff
@@ -131,14 +131,14 @@ Options used:
 To check, type ``docker ps | grep qgis-server`` and you should see a
 line with **qgis-server**::
 
-  CONTAINER ID        IMAGE                 COMMAND                  CREATED             STATUS              PORTS                                            NAMES
-  4de8192da76e        qgis-server           "/tini -- /home/qgis…"   3 seconds ago       Up 2 seconds        0.0.0.0:5555->5555/tcp                           qgis-server
+  CONTAINER ID     IMAGE           COMMAND                  CREATED          STATUS           PORTS                            NAMES
+  4de8192da76e     qgis-server     "/tini -- /home/qgis…"   3 seconds ago    Up 2 seconds     0.0.0.0:5555->5555/tcp           qgis-server
```
Comment (havatv) [inline docs/server_manual/containerized_deployment.rst:135]:
```suggestion
  CONTAINER ID  IMAGE        COMMAND                 CREATED        STATUS        PORTS                   NAMES
  4de8192da76e  qgis-server  "/tini -- /home/qgis…"  3 seconds ago  Up 2 seconds  0.0.0.0:5555->5555/tcp  qgis-server
```

**Review by havatv — APPROVED** [review]
Looks fine. I suggest that we "compact" the sample results even more.

File: docs/server_manual/containerized_deployment.rst (hunk 1 above)
Comment (DelazJ) [reply docs/server_manual/containerized_deployment.rst:135]: I'm not convinced it makes it better: easily/quickly visualize and identify the columns is good for readability. Will be harder if too close. And the change already made the whole table visible.

Comment (havatv) [reply docs/server_manual/containerized_deployment.rst:135]: > I'm not convinced it makes it better: easily/quickly visualize and identify the columns is good for readability. Will be harder if too close. And the change already made the whole table visible.

The whole table was not visible here, and leaving at least two blanks is enough to make it easily readable.

**Comment by DelazJ** [comment]
> The whole table was not visible here

Where is here?

File: docs/server_manual/containerized_deployment.rst (hunk 1 above)
Comment (DelazJ) [reply docs/server_manual/containerized_deployment.rst:135]: > The whole table was not visible here

Where is here?

Comment (havatv) [reply docs/server_manual/containerized_deployment.rst:135]: > > The whole table was not visible here
> 
> Where is here?

In my web browser.

Comment (havatv) [reply docs/server_manual/containerized_deployment.rst]: > Hummm Is it the only one that goes over? Can you check other code frames in that page?
> 
> > leaving at least two blanks is enough to make it easily readable.
> 
> Three blanks are used in https://docs.qgis.org/testing/en/docs/server_manual/containerized_deployment.html#deploying-manifests and I don't think going down is better

I am fine with increasing to three.

**Pull request merged**